README.md

```diff
@@ -141,6 +141,12 @@ The tests can be now ran with `go test`, for example: `go test ./...`.
         }
       }
 
+  responseHeaders:
+    200:
+      Content-Type: ["application/json"]
```

Comment:
Like net/http I made an array. We also could change it to String in the Test but than we have to reformat and concat the Header Values for the keys to one String. It would be easer to check an array and with `IgnoreArraysOrdering` (not added yet) the order doesn't matter.

When we want to use Strings in Test. we could do things like that in the response_header.go:
`
...
		if len(expectedHeader) > 0 {
			respHeaders := make(map[string]string)
			for k, s := range result.ResponseHeaders {
				respHeaders[k] = strings.Join(s,", ")
			}

			errs = append(errs, compare.Compare(expectedHeader, respHeaders, compare.CompareParams{})...)`


?

Reply:
Can we switch to Russian, OK?

Чтобы сделать Gonkey простым и удобным, надо сохранить синтаксис YAML-файла как можно проще. Поэтому я и прошу не требовать массив там, где в 99% случаев он будет состоять из одного элемента.

Склеивание значений в одну строку тоже не нужно, если мы договоримся, что, чтобы тест прошел успешно, серверу достаточно вернуть хотя бы один заголовок, который по имени и значению соответсвует тому, что написано в YAML-файле.

Например, как автор теста, когда я пишу `Accept: application/json`, я предполагаю, что:
- тест упадет, если сервер не вернет заголовок `Accept` или его значение будет не `application/json`
- но при этом тест _не упадет_, если сервер вернет какие-то другие заголовки (мне они не важны в этом тесте)
- тест _не упадет_, если в ответе сервера будет два заголовка `Accept`, при условии, что хотя бы один из них будет иметь значение `application/json`; на значение второго мне наплевать

С одной стороны, это не позволит проверить несколько заголовков Accept. С другой, это упростит синтаксис, а перечисленной функциональности хватит для большинства тестовых случаев. Как давно ты проверял наличие нескольких одинаковых заголовков в ответе сервера?